optimum/amd/ryzenai/modeling.py

```diff
@@ -297,6 +297,9 @@ def _from_pretrained(
 
             vaip_config = provider_options["config_file"]
 
+            if not os.path.exists(vaip_config):
+                raise ValueError(f"Config file not found: {vaip_config}")
```

Comment:
How about loading from the Hub?

Reply:
Yes, this could streamline pre-optimized model usage. I would post a question on the channel for the AMD team to add the vaip configs for the models and we could add support to load directly from hub

Reply:
For other models we would still need this check. But it will be shifted or modified accordingly.

Reply:
Yes I think this is important for usability.

Reply:
In the meantime if you want to merge this, could you maybe just modify the code raising an error that loading from the Hub is not supported / that the user needs to provide the `vaip_config` manually?

Reply:
Makes sense, would update.

Reply:
There already an error message that is printed when the user does not pass a config file. I guess this could clarify the need to pass manually. https://github.com/huggingface/optimum-amd/blob/main/optimum/amd/ryzenai/modeling.py#L285

raise ValueError(
                    "No config file provided. Please provide the necessary config file for" "inference with RyzenAI"
              )

Reply:
oops missed that